packages/langium-vscode/package.json

```diff
@@ -25,13 +25,21 @@
         "scopeName": "source.langium",
         "path": "./data/langium.tmLanguage.json"
       }
+    ],
+    "jsonValidation": [
+      {
+        "fileMatch": "langium-config.json",
+        "url": "./langium-config-schema.json"
+      }
     ]
   },
   "activationEvents": [
     "onLanguage:langium"
   ],
   "main": "out/extension.js",
   "scripts": {
+    "prepare": "npm run copy-schema",
+    "copy-schema": "cp ../langium-cli/langium-config-schema.json ./langium-config-schema.json",
```

Comment:
1. Please move the schema into the existing `data` folder.
2. `cp` won't work on a Windows terminal. Shall we use an npm package for this (like we often use `rimraf` instead of `rm`)?